repl/src/main/scala/ammonite/repl/interp/AmmonitePlugin.scala

```diff
@@ -150,4 +170,49 @@ object AmmonitePlugin{
 
     output(open.toVector)
   }
+}
+
+
+object LineNumberModifier {
+  def apply(g: Global)(unit: g.CompilationUnit,
+                       output: Seq[ImportData] => Unit,
+                       importsLen: => Int) = {
```

Comment:
Let's call this `topWrapperLen` or something, since it's not just the imports but also includes the `object Foo{\n` at the top of the generated file